packages/publish-bundle/node-powers.js

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+/* eslint-env node */
+
+const textEncoder = new TextEncoder();
+const textDecoder = new TextDecoder();
+
+/**
+ * @param {Object} powers
+ * @param {typeof import('http')} powers.http
+ */
+export const makeJsonHttpClient = ({ http }) => {
```

Comment:
I wrote this so it could be faked in tests and to avoid coupling to the powerful `http`.